src/components/gltf-model-plus.js

```diff
@@ -352,7 +359,7 @@ AFRAME.registerComponent("gltf-model-plus", {
         object3DToSet = this.inflatedEl.object3D;
         // TODO: Still don't fully understand the lifecycle here and how it differs between browsers, we should dig in more
         // Wait one tick for the appended custom elements to be connected before attaching templates
-        await nextTick();
+        await AFRAME.scenes[0].systems.ticker.nextTick();
```

Comment:
We should probably use `this.el.sceneEl` instead of `AFRAME.scenes[0]` where we can.